spring-cloud-dataflow-registry/src/test/java/org/springframework/cloud/dataflow/registry/service/DefaultAppRegistryServiceTests.java

```diff
@@ -552,20 +549,6 @@ public void testDeleteAll() throws URISyntaxException {
 		verify(appRegistrationRepository, times(1)).deleteAll(appsToDelete);
 	}
 
-	@Test
-	public void testMultipleBootVersions() {
-		// given
-		Resource resource = new ClassPathResource("AppRegistryTests-importMultipleBootVersions.properties", getClass());
-		// when
-		List<AppRegistration> result = appRegistryService.importAll(false, resource);
-		// then
-		List<AppRegistration> boot2 = result.stream().filter(r -> r.getBootVersion().equals(AppBootSchemaVersion.BOOT2)).collect(Collectors.toList());
-		List<AppRegistration> boot3 = result.stream().filter(r -> r.getBootVersion().equals(AppBootSchemaVersion.BOOT3)).collect(Collectors.toList());
-		assertEquals(1L, boot2.size());
-		assertEquals(1L, boot3.size());
-		assertEquals("2.0.1", boot2.get(0).getVersion());
-		assertEquals("3.0.0", boot3.get(0).getVersion());
-	}
 	@Test
 	public void testMultipleBootVersionsExpectError() {
```

Comment:
I believe this test either is no longer valid OR it needs to be renamed. The corresponding test yml still references the now unsupported `bootVersion` property.